common/sshtools.py

```diff
@@ -42,7 +42,7 @@ class SSH(mount.MountControl):
 
     CHECK_FUSE_GROUP = True
 
-    def __init__(self, cfg = None, profile_id = None, hash_id = None, tmp_mount = False, parent = None, symlink = True, **kwargs):
+    def __init__(self, cfg = None, profile_id = None, hash_id = None, tmp_mount = False, parent = None, symlink = True, read_only = True, **kwargs):
```

Comment:
Ha, yeah, I figured out the kwargs piece after a bit of debugging :) Thanks for confirming my understanding (after a bit of catting /etc/mtab, watching logs, and playing with code).

I've got a bit more to code here, which is why I closed the PR for now - certainly not done yet, as you see :)